tests/core/test_stats.py

```diff
@@ -485,7 +485,7 @@ def test_increment_counter(self):
     def test_increment_counter_with_tags(self):
         self.stats.incr(
             "test_stats_run.delay",
-            tags={"key0": 0, "key1": "val1", "key2": "val2"},
+            tags={"key0": "0", "key1": "val1", "key2": "val2"},
```

Comment:
```suggestion
            tags={"key0": 0, "key1": "val1", "key2": "val2"},
```

If it worked before this test should probably still pass without needing this change.